app/models/dhs1171_pdf.rb

```diff
@@ -39,9 +45,11 @@ def client_data
     }
   end
 
-  def add_cover_sheet_to_completed_form
-    system(
-      "pdftk #{COVERSHEET_PDF} #{completed_filename} cat output #{output_filename}",
-    )
+  def prepend_cover_sheet_to_completed_form
+    system("pdftk #{COVERSHEET_PDF} #{filled_in_form.path} cat output #{complete_form_with_cover.path}")
+  end
+
+  def complete_form_with_cover
+    @_with_cover ||= Tempfile.new(["snap_app_with_cover", ".pdf"], "tmp/")
```

Comment:
this is super nit picky but I almost always create memo-ized ivars with the same name as the method so I never have two ivars with the same name (so here would be `@_completed_form_with_cove` )